libobs/obs-windows.c

```diff
@@ -107,6 +107,18 @@ static void log_processor_cores(void)
 	     os_get_physical_cores(), os_get_logical_cores());
 }
 
+static void log_emulation_status(void)
+{
+#if defined(_M_ARM64EC)
+	blog(LOG_INFO, "Windows ARM64: Compatiable with X64 plugins");
+	return;
+#else
+	if (os_get_emulation_status()) {
```

Comment:
See above, I feel it should always be logged wether OBS is being emulated/translated.